src/styles/index.scss

```diff
@@ -1,3 +1,17 @@
-body {
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+
+@import "./block/card.scss";
+@import "./block/price.scss";
+@import "./block/reviews.scss";
+@import "./block/stars.scss";
+
+* {
   margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

Comment:
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)
3. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)